config/operator/operator_role.yaml

```diff
@@ -129,6 +129,7 @@ rules:
   verbs:
   - get
   - list
+  - watch
```

Comment:
I did test this change, but it did not fix the issue. Are you able to test this change?

Reply:
I did yes, are you certain you deleted the pod after and let it restart? And edited the hive-operator-role?

Reply:
Without running make deploy as that would override a manual change.

Reply:
We'll be able to see if this works in the artifacts collected during the e2e tests.

See, for example, https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_hive/620/pull-ci-openshift-hive-master-e2e/2254/artifacts/e2e/pods/hive_hive-operator-645754b74-xnb6p_hive-operator.log

Reply:
Yes, I did change the operator role and mentioned in the slack chat we had yesterday. I followed this steps to test
1. DEPLOY_IMAGE=quay.io/lalatendum/hive-devel:latest make deploy
2. Did the changes
3. IMG=quay.io/lalatendum/hive-devel:latest make buildah-push
4. `oc delete pods --all -n hive`

When adding watch did not work, I was confused and hence spent almost a whole day debugging it :(

Reply:
The latest e2e run did not have the DeploymentConfig errors.

https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_hive/639/pull-ci-openshift-hive-master-e2e/2261/artifacts/e2e/pods/hive_hive-operator-5db7b64fdb-m9t6l_hive-operator.log